packages/contracts/contracts/router/modules/exchanges/SudoswapModule.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IERC721} from "@openzeppelin/contracts/token/ERC721/IERC721.sol";
+
+import {BaseExchangeModule} from "./BaseExchangeModule.sol";
+import {BaseModule} from "../BaseModule.sol";
+import {ISudoswapRouter} from "../../../interfaces/ISudoswapRouter.sol";
+
+contract SudoswapModule is BaseExchangeModule {
+
+    // --- Fields ---
+
+    ISudoswapRouter public immutable SUDOSWAP_ROUTER;
```

Comment:
We can probably hardcode the Sudoswap router address rather than have it passed in the constructor (to reduce any risks of using the wrong address when deploying).